RealmLoginKit Apple/RealmLoginKit/Controllers/LoginViewController.swift

```diff
@@ -595,17 +595,16 @@ public class LoginViewController: UIViewController, UITableViewDataSource, UITab
         
         saveLoginCredentials()
         
+        var scheme: String?
         var formattedURL = serverURL
         if let schemeRange = formattedURL?.range(of: "://") {
+            scheme = formattedURL?.substring(to: schemeRange.lowerBound)
             formattedURL = formattedURL?.substring(from: schemeRange.upperBound)
         }
         
-        if formattedURL?.range(of: ":") == nil {
-            formattedURL = "\(formattedURL!):9080"
-        }
-        
         let credentials = RLMSyncCredentials(username: email!, password: password!, register: isRegistering)
-        RLMSyncUser.__logIn(with: credentials, authServerURL: URL(string: "http://\(formattedURL!)")!, timeout: 30, onCompletion: { (user, error) in
+        let authScheme = scheme == "realms" ? "https" : "http"
+        RLMSyncUser.__logIn(with: credentials, authServerURL: URL(string: "\(authScheme)://\(formattedURL!)")!, timeout: 30, onCompletion: { (user, error) in
```

Comment:
Since this is the only Realm call, and the framework itself is going to still be used in Objective-C based apps, it didn't make sense to bring in Realm Swift as a dependency. If we need to start adding more Realm logic to this framework, we can review this.